dataactcore/scripts/updateCGAC.py

```diff
@@ -0,0 +1,46 @@
+from dataactcore.models.jobModels import Submission
+from dataactcore.models.userModel import User
+from dataactcore.models.validationInterface import ValidationInterface
+from dataactcore.models.userInterface import UserInterface
+from dataactcore.models.jobTrackerInterface import JobTrackerInterface
+
+def updateCGAC():
```

Comment:
I added it in the repo since it isn't hasn't been run yet. Once it's been successfully PR'd and merged, we'll run the script then remove it from the repo. That was my thinking of the process.